stitching/image_handler.py

```diff
@@ -40,6 +40,11 @@ def set_img_names(self, img_names):
     def resize_to_medium_resolution(self):
         return self.read_and_resize_imgs(self.medium_scaler)
 
+    def to_medium_resolution_binary_mask(self):
+        for img, size in self.input_images():
+            img_medium = self.resize_img_by_scaler(self.medium_scaler, size, img)
+            yield self.binary_img(img_medium)
+
```

Comment:
somehow I don't like img_names and img_sizes being tied to the ImageHandler. I think this should be refactored that the stitcher holds them. Then we wouldn't need to define two Image Handlers with the same arguments

Reply:
I have an idea for this but I need some time to implement it

Reply:
Totally agree. I feel masks should be part of the handler

Reply:
I let you work on this if you have an idea, tell me if you want any proposal/help